Sming/rboot/eagle.rom.addr.v6.ld

```diff
@@ -0,0 +1,347 @@
+PROVIDE ( Cache_Read_Disable = 0x400047f0 );
```

Comment:
Just the latest commit on the feature/rboot branch in raburton/Sming:
```
raburton@debian:~/dev/Sming$ git remote -v
origin	https://raburton@github.com/raburton/Sming.git (fetch)
origin	https://raburton@github.com/raburton/Sming.git (push)
raburton@debian:~/dev/Sming$ git branch -v
* feature/rboot c53fbcf makefile fix closes #278 plus other tweaks
  master        c000e9c actually use SPIFF_BIN_OUT and allow override
```
I notice if you copy and paste the above diff it loses the single space char at the start of each blank line in code, so that might be part of the problem. Could try patch file [here] (https://dl.dropboxusercontent.com/u/5500141/robotiko.diff) instead.